tedana/decay.py

```diff
@@ -37,7 +37,11 @@ def _apply_t2s_floor(t2s, echo_times):
         echo_times = echo_times[:, None]
 
     eps = np.finfo(dtype=t2s.dtype).eps  # smallest value for datatype
-    temp_arr = np.exp(-echo_times / t2s)  # (E x V) array
+    nonzerovox = t2s != 0
+    # Exclude values where t2s is 0 when dividing by t2s.
+    # These voxels are also excluded from bad_voxel_idx
```

Comment:
I think they're included in `bad_voxel_idx` since `temp_arr` would be 0 where `t2s` is zero.

Reply:
Oh never mind I misread the later `t2s != 0`. My bad!